ci-operator/config/openshift/etcd/openshift-etcd-openshift-4.16.yaml

```
@@ -32,6 +32,15 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.16
   to: etcd
+- dockerfile_path: Dockerfile.installer
+  inputs:
+    ocp_4.16_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.16:base-rhel9
+    ocp_builder_rhel-9-etcd-golang-1.19:
```

@r4f4 i think there is a mistake 
```suggestion
    ocp_builder_rhel-9-etcd-golang-1.20:
```

---

Yeah, the etcd build image was replaced by the rhel-9-golang-1.20-openshift-4.16 one. I'll update this once https://github.com/openshift/etcd/pull/251 merges.

---

Fixed.